### ISSUE TITLE
No automatic upload to server on save trigger with pw prompt (when not connected to ftp)

### ISSUE THREAD

**Issue by f34rl3ss** [Equals182/eqFTP#150]
Sorry to bother again ;(
I'm not sure whether this is an expected behaviour or not.
When i start brackets,don't connect to a server via eqFTP and trigger a save, eqFTP asks me about the master password, but nothing more happens.
I would suggest it tries to auto-connect to the server and uploads the file?
When the password was entered, and i trigger a save again, it uploads automatically as it should,
even when not connected to the ftp.
(The auto-upload even on disconnect is checked in the eqFTP server settings)

```
[node-log 12:35:41 PM] [eqFTP-ftpDomain] Loaded
/C:/Users/xxx/AppData/Roaming/Brackets/extensions/user/equals182.eqftp/main.js:1633 [eqFTP] Loaded (v0.7.0)
/C:/Users/xxx/AppData/Roaming/Brackets/extensions/user/equals182.eqftp/main.js:3295 12:35:43.952 [eqFTP-uploadonsave] documentSaved triggered
/C:/Users/xxx/AppData/Roaming/Brackets/extensions/user/equals182.eqftp/main.js:3297 [eqFTP-uploadonsave] Checking if file is within project F:/Files/jic/wp-content/themes/jic/style_jic.css
/C:/Users/xxx/AppData/Roaming/Brackets/extensions/user/equals182.eqftp/main.js:3299 [eqFTP-uploadonsave] Yes it is
/C:/Users/xxx/AppData/Roaming/Brackets/extensions/user/equals182.eqftp/main.js:3301 [eqFTP-uploadonsave] Getting connection ID by path
extensions/default/DebugCommands/NodeDebugUtils.js:117 [node-log 12:35:46 PM] [eqFTP-test] Updating settings. Expecting debug on { debug: true,
  defaultLocal: 'F:/Files/eqFTP Projects' }
extensions/default/DebugCommands/NodeDebugUtils.js:117 [node-log 12:35:46 PM] [eqFTP-ftpDomain]: Adding connections.
extensions/default/DebugCommands/NodeDebugUtils.js:117 [node-log 12:35:46 PM] [eqFTP-ftpDomain]: 136: Checking connection ID: 0
extensions/default/DebugCommands/NodeDebugUtils.js:117 [node-log 12:35:46 PM] [eqFTP-ftpDomain]: 147: Getting existent connection by ID, result: undefined
extensions/default/DebugCommands/NodeDebugUtils.js:117 [node-log 12:35:46 PM] [eqFTP-ftpDomain]: 173: Adding new connection: "Webserver"
```

when i trigger the save again, this happens in the log:

```
12:50:37.297 [eqFTP-uploadonsave] documentSaved triggered
main.js:3297 [eqFTP-uploadonsave] Checking if file is within project F:/Files/jic.li/wp-content/themes/jic/style_jic.css
main.js:3299 [eqFTP-uploadonsave] Yes it is
main.js:3301 [eqFTP-uploadonsave] Getting connection ID by path
main.js:3303 [eqFTP-uploadonsave] Here it is:  0
main.js:3306 [eqFTP-uploadonsave] Automatization type is classic
main.js:3310 [eqFTP-uploadonsave] Upload on save is on, everything is ok
main.js:3323 [eqFTP-uploadonsave] Here's queuer:  
[...]
too much to post, whole connection stuff happening then
```

**Comment by Equals182** [comment]
got it, #148 may have caused it

**Comment by Equals182** [comment]
### Hello there!

I know this issue been there for a pretty long time, but I have some good news here!
We just released eqFTP v**1.0.0-beta.0** (v1-b for short I guess) and we're completely discontinuing support of all previous versions of this plugin.
New version works great on Brackets 1.8 and it should be good on lower versions too.

So yeah, we're closing and locking this issue forever and looking forward for you to try out our new version of eqFTP.
I'm sorry if I somehow disturbed you, or you lost your interest in eqFTP or Brackets in general, we're not going to make any noise here again.

If you have any questions, please contact me via email (you can find it in [my profile](https://github.com/Equals182)) or [Telegram app](http://t.me/equals182)

Thank you for understanding 😊
